_posts/2019-05-02-zarr-2.3-release.md

```diff
@@ -0,0 +1,134 @@
+initial consolidation, reading all metadata and combining them into a
+single object. Once you have done that and deployed the data to a
+cloud object store, the [`open_consolidated()`] function can be used
+to read data, making use of the consolidated metadata.
+
+Support for this new feature is also now available via the
+[xarray](@@TODO) and [intake](@@TODO), and the Pangeo project is using
+consolidated metadata for all new Zarr datasets. E.g., here's an
+example of how to open a Zarr dataset from Pangeo's data catalog via
+intake:
+
+{% highlight python %}
+TODO
+{% endhighlight %}
+
+## Compatibility with N5
+
+Around the same time that development on Zarr was getting started, a
+separate team led by Stephan Saafeld at the Janelia research campus
+was experiencing similar frustrations trying to use HDF5 for storing
+brain imaging data, and developed a software library called
+N5. Although N5 was originally implemented in Java, it has strong
+similarities to Zarr in the approach it takes to storing both metadata
+and data chunks. We became aware of each others' work around a year
+ago and have been learning about the two approaches.
+
+The bottom line is that there is a lot of commonality and we are
+working jointly to bring the two approaches together. As a first
+experimental step towards that goal, the Zarr 2.3 release includes a
+storage adapter which allows reading and writing of data on disk in
+the N5 format. Here's an example:
+
+{% highlight python %}
+TODO
+{% endhighlight %}
+
+## Future developments
+
+There is a growing community of interest around new approaches to
+storage of array-like data, and we'd like to do what we can to build
+connections and share knowledge and ideas between these
+communities. We've started a [regular teleconference](@@TODO) which is
+open to anyone to join, and there is a new [gitter channel](@@TODO)
+for general discussion.
+
+The main focus of our conversations so far has been setting up work
+towards development of a new set of specifications that support the
+features of both Zarr and N5, and the requirements of key user groups,
+including Earth and climate sciences, bioimaging and microscopy, and
+data derived from large scale DNA and RNA sequencing experiments. It
+is still relatively early days and there are lots of open questions to
+work through, both on the technical side and in terms of how we
+organise and coordinate efforts. However, the community is very
+friendly and supportive, and anyone is welcome to participate, so if
+you have an interest please do consider getting involved.
+
+If you would like to stay in touch with where things are going, keep
+an eye on the [zarr](@@TODO) and [zarr-specs](@@TODO) GitHub
+repositories, join the [gitter channel](@@TODO), and please feel free
+to raise issues or add comments if you have any questions or ideas.
+
+## Further reading
+
+* [Zarr Python 2.3 release notes](@@TODO)
+* anything else...
```

Comment:
Should we engage readers (e.g. "We could really use your help with X?")?

Also do we want to mention that a few of us will be at SciPy?

Reply:
> Should we engage readers (e.g. "We could really use your help with X?")?

Good idea, there's lots of things we could use help with but didn't want to overwhelm, not sure what to highlight. I figured engagement in the new spec development is the really big thing, could emphasise that a bit more. Happy to add anything else you think.

> Also do we want to mention that a few of us will be at SciPy?

Good idea!